src/lbaf/Applications/LBAF_app.py

```diff
@@ -112,7 +112,10 @@ def parse_conf_file(self):
         # Parse data parameters if present
         if self.configuration.get("from_data") is not None:
             self.data_stem = self.configuration.get("from_data").get("data_stem")
-            self.phase_ids = self.configuration.get("from_data").get("phase_ids")
+            if isinstance(self.configuration.get("from_data").get("phase_ids"), str):
+                self.phase_ids = list(map(int, self.configuration.get("from_data").get("phase_ids").split('-')))
```

Comment:
Not super familiar with the LBAF code but should this be
```suggestion
                self.phase_ids = range(*map(int, self.configuration.get("from_data").get("phase_ids").split('-')))
```
It looks to me like this will just add the first and last index to the iterations rather than all the indices from the first to the last

Reply:
@nmm0 Wow, I am really sorry! Good there is somebody checking :) - thanks a lot. Not exactly like that, but thanks for the tip, I did too many things at once today :) - sorry! changing to WIP, will be done latest tomorrow!

Reply:
@nmm0 done. Thanks once more! Total blackout today :)